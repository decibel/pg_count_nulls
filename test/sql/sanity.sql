\set ECHO none
\set VERBOSITY verbose

BEGIN;
CREATE EXTENSION count_nulls;

-- Remember that JSON only accepts 'null'
SELECT null_count('{"a": null}'::jsonb);
SELECT null_count(1,NULL);

\echo TRANSACTION INTENTIONALLY LEFT OPEN
