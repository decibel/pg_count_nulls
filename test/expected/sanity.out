\set ECHO none
 null_count 
------------
          1
(1 row)

 null_count 
------------
          1
(1 row)

TRANSACTION INTENTIONALLY LEFT OPEN
